src/styles/dotcms-theme/components/_calendar.scss

```diff
@@ -1,261 +1,310 @@
-  border-color: transparent;
-  background: $bg-color-secondary;
+    color: $hover-text-color;
+    border-color: transparent;
+    background: $hover-bg;
 }
+
 .p-datepicker .p-datepicker-header .p-datepicker-prev:focus,
 .p-datepicker .p-datepicker-header .p-datepicker-next:focus {
-  outline: 0 none;
-  outline-offset: 0;
-  box-shadow: none;
+    outline: 0 none;
+    outline-offset: 0;
+    box-shadow: none;
 }
+
 .p-datepicker .p-datepicker-header .p-datepicker-title {
-  line-height: $spacing-5;
+    line-height: $spacing-5;
 }
+
 .p-datepicker .p-datepicker-header .p-datepicker-title select {
-  transition: background-color 0.2s, border-color 0.2s, color 0.2s, box-shadow 0.2s, background-size 0.2s cubic-bezier(0.64, 0.09, 0.08, 1);
+    transition: background-color 0.2s, border-color 0.2s, color 0.2s, box-shadow 0.2s, background-size 0.2s cubic-bezier(0.64, 0.09, 0.08, 1);
 }
+
 .p-datepicker .p-datepicker-header .p-datepicker-title select:focus {
-  outline: 0 none;
-  outline-offset: 0;
-  box-shadow: none;
-  border-color:  $brand-primary;
+    outline: 0 none;
+    outline-offset: 0;
+    box-shadow: none;
+    border-color: $brand-primary;
 }
+
 .p-datepicker .p-datepicker-header .p-datepicker-title .p-datepicker-month {
-  margin-right: $spacing-1;
+    margin-right: $spacing-1;
 }
+
 .p-datepicker table {
-  font-size: $spacing-3;
-  margin: $spacing-1 0;
+    font-size: $spacing-3;
+    margin: $spacing-1 0;
 }
+
 .p-datepicker table th {
-  padding: $spacing-1;
+    padding: $spacing-1;
 }
+
 .p-datepicker table th > span {
-  width: $spacing-7;
-  height: $spacing-7;
+    width: $spacing-7;
+    height: $spacing-7;
 }
+
 .p-datepicker table td {
-  padding: $spacing-1;
+    padding: $spacing-1;
 }
+
 .p-datepicker table td > span {
-  width: $spacing-7;
-  height: $spacing-7;
-  border-radius: 50%;
-  transition: none;
-  border: 1px solid transparent;
+    width: $spacing-7;
+    height: $spacing-7;
+    border-radius: 50%;
+    transition: none;
+    border: 1px solid transparent;
 }
+
 .p-datepicker table td > span.p-highlight {
-  color:  $highlight-text-color;
-  background: $highlight-bg;
+    color: $highlight-text-color;
+    background: $highlight-bg;
 }
+
 .p-datepicker table td > span:focus {
-  outline: 0 none;
-  outline-offset: 0;
-  box-shadow: none;
+    outline: 0 none;
+    outline-offset: 0;
+    box-shadow: none;
 }
+
 .p-datepicker table td.p-datepicker-today > span {
-  background: $white;
-  color: $black;
-  border-color: $gray-light;
+    background: $white;
+    color: $black;
+    border-color: $gray-light;
 }
+
 .p-datepicker table td.p-datepicker-today > span.p-highlight {
-  color:  $highlight-text-color;
-  background: $highlight-bg;
+    color: $highlight-text-color;
+    background: $highlight-bg;
 }
+
 .p-datepicker .p-datepicker-buttonbar {
-  padding: $spacing-2 0;
-  border-top: 1px solid $bg-color-secondary;
+    padding: $spacing-2 0;
+    border-top: 1px solid $bg-color-secondary;
```

Comment:
again, bg-color in a border, maybe is the name of the variable, we might have to review it.

Reply:
This is for easier search & replace. Not final.